package.json

```diff
@@ -50,6 +49,7 @@
     "pem": "^1.14.2",
     "prettier": "^1.18.2",
     "pretty-quick": "^1.11.1",
+    "qss": "^2.0.3",
```

Comment:
Do we still need qss then?

Reply:
`qss` is still used as a dev dependency for integration tests.